### PR TITLE
Ensure that aws profiles are getting cleared/deleted after each build completion

### PR DESCRIPTION
**Pull request by rajeevr2715**
### Requirements

Ensure aws profiles are getting cleared/deleted after each build completion

### Description of the change

Currently we have issues with builds as the aws profiles are not cleared after build completion.

Possible reasons for the build issues:
1. We’ll need to clear the profile (reset keys) after every build.  It might mean that the change to use a new random profile name with every build had a bug (not clearing our profiles/keys).
2.  Clearing just the keys would still leave the profile (with dummy keys) in the creds file. This will make the file larger & larger. We’ll need to find a way to remove the profile  after the build instead of just clearing out the keys.

This change deletes the aws profile (references) after the builds gets completed (success/fail).

### Possible Drawbacks

NA

### Applicable Issues

NA

### CODE REVIEW

**Comment by rajeevr2715** [comment]
Done

**Comment by rajeevr2715** [comment]
@SiniWilson cloud-api profile is not random, so removing this from file might cause issue in parallel builds.

File: builds/delete-serverless-service-build-pack/Jenkinsfile [hunk 1]
```diff
@@ -850,6 +850,8 @@ def resetCredentials(credsId) {
 	echo "resetting AWS credentials"
 	sh "aws configure set profile.${credsId}.aws_access_key_id XXXXXXXXXXXXXXXXXXXXXXXXXX"
 	sh "aws configure set profile.${credsId}.aws_secret_access_key XXXXXXXXXXXXXXXXXXXXXX"
```
Comment (pspani) [inline builds/delete-serverless-service-build-pack/Jenkinsfile]: if we are deleting the creds, then replacing the creds with XXXXX seems redundant

Comment (rajeevr2715) [reply builds/delete-serverless-service-build-pack/Jenkinsfile]: yeah, will remove

Comment (rajeevr2715) [reply builds/delete-serverless-service-build-pack/Jenkinsfile]: Done

File: builds/delete-serverless-service-build-pack/Jenkinsfile [hunk 2]
```diff
@@ -848,8 +848,8 @@ def cleanupS3BucketPolicy(stage, assetInfo) {
  */
 def resetCredentials(credsId) {
 	echo "resetting AWS credentials"
-	sh "aws configure set profile.${credsId}.aws_access_key_id XXXXXXXXXXXXXXXXXXXXXXXXXX"
-	sh "aws configure set profile.${credsId}.aws_secret_access_key XXXXXXXXXXXXXXXXXXXXXX"
+  def credPath = System.getenv().HOME + "/.aws/credentials"
+	sh "sed -i '/${credsId}/,+2d' ${credPath}"
```
Comment (devsatishm) [inline builds/delete-serverless-service-build-pack/Jenkinsfile]: @rajeevr2715  Can you check if we'll need to do something similar in  ~/.aws/config?

Comment (rajeevr2715) [reply builds/delete-serverless-service-build-pack/Jenkinsfile]: Yes. removed that also.

**Pull request merged**